### PR TITLE
[#245] Add to Configuration step Usage section

### LINKED ISSUE #245: Add to Configuration step Usage section
Usage section (https://kcjh3g.axshare.com/#g=1&p=request-step-2-dodatkowe_info) contain 3 elements:
- customer typology (select: single user; representing a research community/project; representing a private company)
- reason to access (text area)
- additional comments (text area)

EDIT: part of #228

### PR DESCRIPTION
**Pull request by martaswiatkowska**
Fixed #245

### CODE REVIEW

File: app/views/services/configurations/show.html.haml [hunk 1]
```diff
@@ -3,7 +3,9 @@ Configuration
 = simple_form_for @project_item, url: service_configuration_path(@service),
   method: :put, html: { id: "order-form"  } do |f|
   = f.association :project, collection: @projects
-
+  = f.input :customer_typology, collection: ProjectItem.customer_typologies.keys
```
Comment (mkasztelnik) [inline app/views/services/configurations/show.html.haml]: `f.association` from simple form should be used here.

File: db/migrate/20181025095231_add_usage_section_to_project_item.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AddUsageSectionToProjectItem < ActiveRecord::Migration[5.2]
+  def change
+    add_column :project_items, :customer_typology, :string
```
Comment (mkasztelnik) [inline db/migrate/20181025095231_add_usage_section_to_project_item.rb]: String or int?

File: spec/factories/project_items.rb [hunk 3]
```diff
@@ -5,5 +5,8 @@
     status :created
     offer
     project
+    customer_typology { |n| ProjectItem.customer_typologies.keys.sample }
```
Comment (mkasztelnik) [inline spec/factories/project_items.rb]: if `|n|` is not used it can be removed.

File: spec/features/order_spec.rb [hunk 4]
```diff
@@ -52,12 +52,20 @@
                                     "Next", exact: true)
 
       select "Services"
+      select "Single user", from: "Customer typology"
+      fill_in "Access reason", with: "To pass test"
+      fill_in "Additional information", with: "Additional information test"
+
+      sleep(1)
```
Comment (mkasztelnik) [inline spec/features/order_spec.rb]: Don't use `sleep(1)`!!!!

Comment (martaswiatkowska) [reply spec/features/order_spec.rb]: Sory! I didn't clean it, it was'n purpose

File: db/migrate/20181025095231_add_usage_section_to_project_item.rb (hunk 2 above)
Comment (martaswiatkowska) [reply db/migrate/20181025095231_add_usage_section_to_project_item.rb]: String, I'm saving all string in db

**Comment by roksanaer** [comment]
tested in the deployed version - usage sections works well (however as @mkasztelnik noticed it lacks affiliation but it wasn't in the scope of this one - @abacz we must add it). additionaly Other info field is not mandatory

**Pull request merged**